sdk/core/azure-core/test/ut/string_test.cpp

```diff
@@ -23,18 +25,20 @@ TEST(String, invariantCompare)
 TEST(String, toLowerC)
 {
   using Azure::Core::_internal::StringExtensions;
-  for (unsigned char ch = 0; ch < 255; ch += 1)
+  for (unsigned i = 0; i <= 255; ++i)
```

Comment:
What's the type of `i`? When not specified, is it just a 32-bit unsigned `int`?

Reply:
Yes - in C (and C++), the "unsigned" type is a shorthand for "unsigned int".